packages/golden-layout/.gitignore

```diff
@@ -1,6 +1,3 @@
-node_modules
 .idea
 npm-debug.log
```

Comment:
These shouldn't be needed here

Side note, add `.idea` to the root gitignore in case someone opens the project in IntelliJ or Webstorm. That's the Jetbrains project settings folder similar to `.vscode`